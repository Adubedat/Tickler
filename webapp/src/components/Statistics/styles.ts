import styled from 'styled-components';

export const StatisticsTitle = styled.span`
  font-size: 2.5em;
  font-weight: 500;
  color: #008aa8;
`;

export const StatisticsContainer = styled.div`
  position: relative;
  display: flex;
  flex-direction: column;
  padding: 20px;
  flex: 5;
`;
