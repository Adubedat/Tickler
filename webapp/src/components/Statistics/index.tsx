import React from 'react';
import { StatisticsContainer, StatisticsTitle } from './styles'

export const Statistics = () => {
  return (
    <StatisticsContainer>
      <StatisticsTitle>Statistics</StatisticsTitle>
    </StatisticsContainer>
  );
};

export default Statistics;
