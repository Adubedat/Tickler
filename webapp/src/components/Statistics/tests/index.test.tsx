import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Statistics from "../index";

describe("<Statistics />", () => {
  test("renders", () => {
    const wrapper = shallow(<Statistics />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});