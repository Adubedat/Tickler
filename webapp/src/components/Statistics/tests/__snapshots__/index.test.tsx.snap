// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Statistics /> renders 1`] = `
<styled.div>
  <styled.span>
    Statistics
  </styled.span>
</styled.div>
`;
